tests/beaker_tests/cisco_vlan/vlan/standardvlan_provider_nondefaults.rb

```diff
@@ -99,10 +99,11 @@
       "resource cisco_vlan '128'", options)
     on(agent, cmd_str) do
       search_pattern_in_output(stdout,
-                               { 'ensure'    => 'present',
-                                 'shutdown'  => 'true',
-                                 'state'     => 'suspend',
-                                 'vlan_name' => 'DESCR-VLAN0128' },
+                               { 'ensure'     => 'present',
+                                 'mapped_vni' => '128000',
```

Comment:
I am trying to use 'platform' method in file standardvlan_provider_nondefaults.rb. It gives a runtime error. I have synced to latest release_1.2.0. Looks like platform returned was " ". Am I missing something?

usage:
 plat = platform.match('n9k')

TestStep :: Check cisco_vlan resource presence on agent
    
    agent1 19:47:06$ sudo ip netns exec management /opt/puppetlabs/bin/facter -p cisco.hardware.type

    
    agent1 executed in 4.80 seconds
    #<RuntimeError: Unrecognized platform type: 
    >
    /home/rtpfe1/aloaggar/pnc/cisco-network-puppet-module/tests/beaker_tests/lib/utilitylib.rb:724
    /home/rtpfe1/aloaggar/pnc/cisco-network-puppet-module/tests/beaker_tests/cisco_vlan/vlan/standardvlan_provider_nondefaults.rb:100
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/dsl/structure.rb:43
    /home/rtpfe1/aloaggar/pnc/cisco-network-puppet-module/tests/beaker_tests/cisco_vlan/vlan/standardvlan_provider_nondefaults.rb:95
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/dsl/structure.rb:58
    /home/rtpfe1/aloaggar/pnc/cisco-network-puppet-module/tests/beaker_tests/cisco_vlan/vlan/standardvlan_provider_nondefaults.rb:66
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/test_case.rb:128
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/test_case.rb:128
    /home/rtpfe1/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/benchmark.rb:303
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/test_case.rb:125
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/test_suite.rb:311
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/test_suite.rb:308
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/test_suite.rb:308
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/test_suite.rb:357
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/cli.rb:159
    /home/rtpfe1/.rvm/gems/ruby-2.2.1/gems/beaker-2.26.0/lib/beaker/cli.rb:99